assembl/static2/js/app/pages/brightMirrorFiction.jsx

```diff
@@ -6,91 +6,207 @@ import { I18n } from 'react-redux-i18n';
 // Graphql imports
 import { compose, graphql } from 'react-apollo';
 import BrightMirrorFictionQuery from '../graphql/BrightMirrorFictionQuery.graphql';
+// Optimization: Should create createBrightMirrorComment.graphql and adapt the mutation
+import CreateBrightMirrorCommentMutation from '../graphql/mutations/createPost.graphql';
+// Optimization: Should create ideaWithCommentsQuery.graphql and adapt the query
+import IdeaWithCommentsQuery from '../graphql/IdeaWithPostsQuery.graphql';
```

Comment:
with this query you retrieve all the comments. it's not good for perf. For the Tree component, we only need Post ids.

Reply:
Cf. comment above - Optimization: Should create ideaWithCommentsQuery.graphql and adapt the query.

Reply:
it's just a comment! implements it please